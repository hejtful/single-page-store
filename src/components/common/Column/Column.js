import React from 'react';

import styles from './Column.module.css';

export const Column = ({ children }) => (
  <section className={styles.column}>{children}</section>
);
