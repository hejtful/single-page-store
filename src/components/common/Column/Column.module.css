.column {
  border: 1px solid #dbdbdb;
  padding: 1em;
}
