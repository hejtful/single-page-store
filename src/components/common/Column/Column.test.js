import React from 'react';
import { render } from '@testing-library/react';
import { Column } from './Column';

it('renders column element and all children', () => {
  const { container } = render(
    <Column>
      <div>Test child 1</div>
      <div>Test child 2</div>
    </Column>
  );

  expect(container).toMatchSnapshot();
});
