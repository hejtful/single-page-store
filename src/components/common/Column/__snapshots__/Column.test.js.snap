// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders column element and all children 1`] = `
<div>
  <section
    class="column"
  >
    <div>
      Test child 1
    </div>
    <div>
      Test child 2
    </div>
  </section>
</div>
`;
